tests/test_core.py

```diff
@@ -1762,7 +1761,6 @@ def test_bigswitch(self):
 
   @no_emterpreter
   def test_biggerswitch(self):
```

Comment:
I see that `binaryen2.test_biggerswitch` is kind of slow to compile to bitcode, yeah (30 seconds). otherwise `binaryen0` or `bigswitch` seem ok.

i can disable these tests for now in the wasm backend if you want?